src/lib/containers/works_for_you.tsx

```diff
@@ -128,6 +131,10 @@ export class WorksForYou extends React.Component<Props, State> {
     })
   }
 
+  componentDidUpdate() {
+    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: false })
```

Comment:
If you don’t scroll at all `this.currentScrollOffset` will be `undefined`, right? Does it not break when doing `undefined + 1`?